src/Endpoints/Resolve.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace FiveamCode\LaravelNotionApi\Endpoints;
+
+use FiveamCode\LaravelNotionApi\Entities\Blocks\Block;
+use FiveamCode\LaravelNotionApi\Entities\Database;
+use FiveamCode\LaravelNotionApi\Entities\NotionParent;
+use FiveamCode\LaravelNotionApi\Entities\Page;
+use FiveamCode\LaravelNotionApi\Entities\User;
+use FiveamCode\LaravelNotionApi\Exceptions\HandlingException;
+use FiveamCode\LaravelNotionApi\Exceptions\NotionException;
+use FiveamCode\LaravelNotionApi\Notion;
+
+/**
+ * Class Resolve.
+ */
+class Resolve extends Endpoint
+{
+    /**
+     * Block constructor.
+     *
+     * @param  Notion  $notion
+     *
+     * @throws HandlingException
+     * @throws \FiveamCode\LaravelNotionApi\Exceptions\LaravelNotionAPIException
```

Comment:
The exception is not correct here, the parent constructor in `Endpoint` throws a `HandlingException`. So I think we can get rid of that one (Just noticed that, it's also already wrong in the other endpoints)